Framework/test/testWorkflow.cxx

```diff
@@ -0,0 +1,103 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
This test runs a DPL workflow with full QC chain and returns 0 if it works correctly (they do it in a similar way in O2). To make it behave properly, we need a timeout of around 15-30 seconds - it might not exit automatically when something is wrong. Also, it needs to be ran on machine with glfw, without ssh and docker, or with '-b' flag. Is this the case already for QC?

Reply:
I am not sure to understand the uqestion. Let's discuss Monday.

Reply:
The test testWorkflow took 1.69 seconds on my Mac.